tests/net/utils/src/main.c

```diff
@@ -108,69 +108,6 @@ static const unsigned char pkt3[199] = {
 0x90, 0x91, 0x92, 0x93, 0x94, 0x95, 0x96        /* ....... */
 };
 
-/* IPv6 TCP (224 bytes) */
-static const unsigned char v6_tcp_pkt1[224] = {
-/* IPv6 header starts here */
-0x60, 0x00, /* .J....`. */
```

Comment:
net_pkt_get_src/dst disappearing there is no need for any test

Reply:
That test was relevant because of the existence of net_pkt_get_src/dst_addr functions. But these are removed. (and should never come back actually, or that will mean we do something wrong in the core).